src/rpc/index.js

```diff
@@ -32,12 +32,15 @@ class RPC {
     this.pendingAcks = [];
     this.messagesWaitAuth = [];
     this.messagesWaitResponse = new Map();
+    this.handleTransportOpen = this.handleTransportOpen.bind(this)
+    this.handleTransportError = this.handleTransportError.bind(this)
+    this.handleTransportMessage = this.handleTransportMessage.bind(this)
```

Comment:
Please stick to the style. Add a commas and semicolons

Reply:
fixed here https://github.com/alik0211/mtproto-core/pull/235/commits/185d52e3ffbdc65bcfe7b69d810a0b4a4ff87446